src/ctrs_sprintf.c

```diff
@@ -17,7 +17,7 @@ static char* ctrs_vsprintf_char(const char* format, va_list va)
     int neededSize = vsnprintf(NULL, 0, format, va);
     if (neededSize < 0)
     {
-        LogError("failure in vsnprintf(NULL, 0, format=%s, va=%p)", format, (void*)va);
+        LogError("failure in vsnprintf(NULL, 0, format=%s, va=%p)", format, (void*)&va);
```

Comment:
>&va [](http://example.com/codeflow?start=83&length=3)

Interesting. How come this was not caught in our Linux build? We also build with gcc ...

Reply:
Good question. Not sure. I ran into this on Ubuntu 20.04 with the latest bits of everything. I don't seem to be able to see the DevOps pipeline run for this and earlier PRs for some reason. 🤷🏽‍♂️

Reply:
I take that back. I _am_ able to see the pipeline runs. The [Linux build](https://msazure.visualstudio.com/One/_build/results?buildId=43309674&view=logs&j=ae9a2f62-3ee1-5984-579f-87b38be696e5) seems to be on Ubuntu 18.04 and I was running 20.04. Maybe newer GCC is less forgiving.